src/boards/memmap_default_rp2350.ld

```diff
@@ -29,6 +29,7 @@ MEMORY
     INCLUDE "pico_flash_region.ld"
     */
     FLASH(rx) : ORIGIN = 0x10000000, LENGTH = 16384k
+    PSRAM(rwx) : ORIGIN = 0x11000000, LENGTH = 8192k
```

Comment:
What result when no PSRAM exists or is used in the build?   Should this be distinct memmap for PSRAM enabled boards?   (I really am unsure of best practice here.)